packages/components/src/components/SquiggleChart.tsx

```diff
@@ -105,7 +105,11 @@ const SquiggleItem: React.FC<SquiggleItemProps> = ({
       return (
         <VariableBox heading="Array">
           {expression.value.map((r) => (
-            <SquiggleItem expression={r} width={width - 20} height={50} />
+            <SquiggleItem
+              expression={r}
+              width={width ? width - 20 : width}
```

Comment:
This line is confusing. Maybe ``{_.isFinite(width) ? width - 20 : false}`` or something